### PR TITLE
Increase KubeAPIQPS and KubeAPIBurst to 50/100 respectively

### PR DESCRIPTION
**Pull request by RobertKrawitz**
**- What I did**

Increase the kubelet API QPS parameters to allow for more API transactions by the kubelet.

**- How to verify it**

Inspect the kubelet.conf on workers and masters, and the kubelet logs to ensure that the parameters are set accordingly

**- Description for the changelog**

Increase the kubelet API QPS parameters to allow for heavier load on nodes.

### CODE REVIEW

**Comment by RobertKrawitz** [comment]
@rphillips

**Comment by rphillips** [comment]
The 'k' is likely lowercased in the yaml file

**Comment by rphillips** [comment]
otherwise, lgtm

**Comment by RobertKrawitz** [comment]
I'm pretty sure the parameters are upper case (that's what I've been testing with at any rate and it has taken effect).

File: templates/master/01-master-kubelet/_base/files/kubelet.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ contents:
     clusterDomain: cluster.local
     containerLogMaxSize: 50Mi
     maxPods: 250
+    KubeAPIQPS: 50
```
Comment (rphillips) [inline templates/master/01-master-kubelet/_base/files/kubelet.yaml]: `K` needs an update in this file as well.

**Comment by rphillips** [comment]
until the config gets resolved.

/hold

**Comment by RobertKrawitz** [comment]
> until the config gets resolved.
> 
> /hold
Fixed.

**Comment by rphillips** [comment]
vpc limits exceeded

**Pull request merged**